### PR TITLE
Explain the Action module

### PR DESCRIPTION
**Pull request by ylei-tsubame**
Add documentation for the added Action module.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,96 @@
...
-* It is extremely easy to use, minimizing the opportunity for human error and making it as simple as possible for "remote hands" to affect repairs and replacements without any prior availability experience and minimal technical knowledge.
+- It is the first server platform designed with the singular focus of protecting your servers and keeping them running under even extreme fault conditions.
+- It is fully self-contained, making it ideal for totally offline operation.
+- It is a "self driving" server availability platform that can continuously monitors internal and external data sources, compiling, analyzing and autonomously deciding when and what action to take to protect your servers. It is ideally suited for extended remote deployments and "hands off" operation.
+- It is based on an extensively field tested, open architecture with full data, mechanical and electrical redundancy allowing any component to be failed, removed and replaced without the need for a maintenance window. The _ANvil!_ platform has over five years of real-world deployment over dozens of sites and an historic uptime of over 99.9999%.
```
Comment (manekenpix) [inline README.md:10]:
```
...maintenance window. The _ANvil!_ platform has over...
```
That `n` in _ANvil!_ should be small case?

Comment (manekenpix) [inline README.md]:
```
...maintenance window. The _ANvil!_ platform has over...
```
That `n` in _ANvil!_ should be small case?

File: README.md [hunk 2]
```diff
@@ -1,23 +1,96 @@
...
 In short, it is a server platform that just won't die.
 
-* How do you build an *Anvil!*?
+- How do you build an _Anvil!_?
 
 It's quite easy, but it does require a little more space than a README allows for.
-* [How to Build an m2 Anvil!](https://www.alteeve.com/w/Build_an_m2_Anvil!)
 
-The *Anvil!* was designed and extensively tested on [Primergy](http://www.fujitsu.com/global/products/computing/servers/primergy/) servers, Brocade [ICX](http://www.brocade.com/en/products-services/switches/campus-network-switches.html) switches and APC [SmartUPS](http://www.apc.com/smartups/index.cfm?ISOCountryCode=ca) UPSes and [Switched PDU](http://www.apc.com/shop/ca/en/categories/power-distribution/rack-power-distribution/switched-rack-pdu/_/N-17k76am)s. That said, the *Anvil!* platform is designed to be hardware agnostic and should work just fine on Dell, Cisco USC, NEC, Lenovo x-series, and other tier-1 server vendors. 
+- [How to Build an m2 Anvil!](https://www.alteeve.com/w/Build_an_m2_Anvil!)
```
Comment (manekenpix) [inline README.md]: _Anvil!_ is in italics everywhere, why not here?

File: README.md [hunk 3]
```diff
@@ -1,23 +1,96 @@
-[Anvil!](https://www.alteeve.com/w/Anvil!) m2 - [Striker](https://www.alteeve.com/w/Striker) + [ScanCore](https://www.alteeve.com/w/ScanCore) v2.0.1
-=============
+# [Anvil!](https://www.alteeve.com/w/Anvil!) m2 - [Striker](https://www.alteeve.com/w/Striker) + [ScanCore](https://www.alteeve.com/w/ScanCore) v2.0.1
 
-Welcome to the v2.0.0 release of the *Anvil!* m2 [Intelligent Availability](https://www.alteeve.com/w/Intelligent_Availability)™ platform! 
+Welcome to the v2.0.0 release of the _Anvil!_ m2 [Intelligent Availability](https://www.alteeve.com/w/Intelligent_Availability)™ platform!
 
-What is an *Anvil!* Platform?
+What is an _Anvil!_ Platform?
```
Comment (manekenpix) [inline README.md]: Does `Platform` need capital P here?

File: README.md [hunk 4]
```diff
@@ -1,23 +1,96 @@
...
+- [How to Build an m2 Anvil!](https://www.alteeve.com/w/Build_an_m2_Anvil!)
 
-[Alteeve](https://www.alteeve.com/), the company behind the *Anvil!* project, actively supports the [open source](https://www.alteeve.com/w/Support) community. We also offer commercial support contracts to assist with any stage of deployment, operation and custom development.
+The _Anvil!_ was designed and extensively tested on [Primergy](http://www.fujitsu.com/global/products/computing/servers/primergy/) servers, Brocade [ICX](http://www.brocade.com/en/products-services/switches/campus-network-switches.html) switches and APC [SmartUPS](http://www.apc.com/smartups/index.cfm?ISOCountryCode=ca) UPSes and [Switched PDU](http://www.apc.com/shop/ca/en/categories/power-distribution/rack-power-distribution/switched-rack-pdu/_/N-17k76am)s. That said, the _Anvil!_ platform is designed to be hardware agnostic and should work just fine on Dell, Cisco USC, NEC, Lenovo x-series, and other tier-1 server vendors.
```
Comment (manekenpix) [inline README.md]: At the beginning of this paragraph:
```
The Anvil! was designed and...
```
I think you could say 
```
The Anvil! platform was designed and...
```
like everywhere else.

File: README.md [hunk 5]
```diff
@@ -1,23 +1,96 @@
...
+
+### Node Decisions
+
+These decisions are only made by **nodes**.
+
+| Number | Definition                                                                                                                                                |
+| ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 1      | Enter (or reenter) stable **power** state and clear power Warnings.                                                                                       |
+| 2      | Enter (or reenter) stable **temperature** state and clear temperature Warnings.                                                                           |
+| 3      | Enter **power** Warning state and send notifications stating that power abnormalities were detected.                                                      |
```
Comment (manekenpix) [inline README.md]: I think the `w` in `Warning` should be small case.

File: README.md [hunk 6]
```diff
@@ -1,23 +1,96 @@
...
+| 4      | Enter **temperature** Warning state and send notifications stating that temperature abnormalities were detected.                                          |
```
Comment (manekenpix) [inline README.md]: Same as above

**Review by manekenpix — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

File: README.md (hunk 2 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

File: README.md (hunk 3 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

File: README.md (hunk 4 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

File: README.md (hunk 5 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

File: README.md (hunk 6 above)
Comment (ylei-tsubame) [reply README.md]: Changed as suggested.

**Review by manekenpix — APPROVED** [review]

**Pull request merged**